### PR TITLE
Fix IntegrateEllipsoidsTwoStep always scaling by bin width

### LINKED ISSUE #21106: IntegrateEllipsoidsTwoStep intensities should be consistent with IntegratePeaksMD
`IntegrateEllipsoidsTwoStep` returns an intensity value that is differs from `IntegratePeaksMD`  because `IntegrateEllipsoidsTwoStep` uses counts but `ConvertToMD` prior to `IntegratePeaksMD` will use either frequency **or** counts depending on the state of the workspace.

### Expected behavior
Integrated intensities output from `IntegrateEllipsoids` and `IntegratePeaksMD` should be consistent.

### Actual behavior
`IntegrateEllipsoids` intensities are always divided by the bin width, while `IntegratePeaksMD` will vary according depending on the workspace type.

### Steps to reproduce the behavior
Here's a script that reproduces the behaviour

```python
ps = mtd["38868_ruby_peaks"]
ws = mtd["38868_ruby_vana_processed"]

md =ConvertToDiffractionMDWorkspace(ws, OneEventPerBin=False)

region_radius = 0.2
bkg_inner = 0.18
bkg_outer = 0.2
peak_radius = 0.18

spherical_integration = IntegratePeaksMD(md, ps, PeakRadius=peak_radius, CorrectIfOnEdge=False, IntegrateIfOnEdge=True, BackgroundInnerRadius=bkg_inner, BackgroundOuterRadius=bkg_outer, UseOnePercentBackgroundCorrection=False)
elliptical_integration = IntegrateEllipsoidsTwoStep(ws, ps, RegionRadius=region_radius, PeakSize=peak_radius, BackgroundInnerSize=bkg_inner, BackgroundOuterSize=bkg_outer, UseOnePercentBackgroundCorrection=False)
```
You can compare by overlaying both on the slice viewer and inspecting a peak

Data files can be found on /olympic/bablylon5/scratch/Sam

### Platforms affected
All

### PR DESCRIPTION
**Pull request by samueljackson92**
This makes the integrated intensities consistent with the values used in MD workspaces output from `ConvertToDiffractionMDWorkspace`.

**To test:**
See the bug report for how to reproduce the issue. There is also information on where to get data files.

Fixes #21106

**Release Notes** 
[See here](https://github.com/mantidproject/mantid/pull/21109/commits/8f882be382244970a1021c5b1eb3129a9c7fb959)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v3.12.0/diffraction.rst [hunk 1]
```diff
@@ -41,6 +41,9 @@ Engineering Diffraction
 Single Crystal Diffraction
 --------------------------
 - :ref:`FilterPeaks <algm-FilterPeaks>` now supports filtering peaks by TOF, d-spacing, and wavelength.
+- HB3A reduction interface has been enhanced.  A child window is added to it for users to pre-process scans and save the processed and merged data to NeXus files in order to save time when they start to reduce and visualize the data.
```
Comment (josephframsay) [inline docs/source/release/v3.12.0/diffraction.rst]: Is this saying the same thing as line 48?

**Review by josephframsay — CHANGES_REQUESTED** [review]
Changes are nice, and the integration values now appear to be correct. Just one small question about the docs

File: docs/source/release/v3.12.0/diffraction.rst (hunk 1 above)
Comment (samueljackson92) [reply docs/source/release/v3.12.0/diffraction.rst]: I think you are correct. Probably came in when merging. I will remove the duplicate.

**Review by josephframsay — APPROVED** [review]

**Comment by martyngigg** [comment]
Final fix was a docs-only change so no need to wait for the system tests.

**Pull request merged**